org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/EObjectSnapshotProvider.java

```diff
@@ -236,8 +240,8 @@ public String toString() {
 
 	}
 
-	public IResourceSnapshot createResourceSnapshot(Resource resource) {
-		return new ResourceSnapshot(this, resource);
+	public IResourceSnapshot createResourceSnapshot(Resource resource, boolean recordReferences) {
+		return new ResourceSnapshot(this, resource, recordReferences);
```

Comment:
Add an overload without the new parameter which defaults to true for compatibility?

Reply:
do we need that.afaik this is not yet api

Reply:
this is indeed not API... skipping.